idaplugin/rematch/updater.py

```diff
@@ -18,6 +18,11 @@ def update():
   if not config['settings']['update']['autocheck']:
     return
 
+  if not config['login']['server']:
+    logger('updater').debug("skipping auto update because server is\
+                            not configured yet, is it your first time ?")
+    return
+
```

Comment:
`config['login']['server']` is not required by the update code. Why is this condition needed?